examples/multiple_test_pages/tests.js

```diff
@@ -0,0 +1,11 @@
+for (var i = 0; i < 200; i++) {
+  if (window.location.search === '?batch=1') {
```

Comment:
No, that would require testem to have intimate knowledge of how your code is built and run once it is loaded into the browser. This feature is more concerned with giving users the ability to use more than one test page, what they do with those test pages is up to them.